templates/legal/ubuntu-advantage-service-description/index.md

```diff
@@ -0,0 +1,117 @@
+        2.  Access to the Landscape SaaS systems management tool is included with all support offerings, unless otherwise noted.
+
+3.  <h2 id="uasd-severity-levels">Severity levels and target response times</h2>
+
+    - Once a support request is opened, a Canonical Support Engineer will validate the case information and determine the severity level, working with the customer to assess the urgency of the case.
+    - Response times will be as set forth in the Service Description for the applicable service offering.
+    - When setting the severity level, Canonical's Support Team will use the definitions as stated below:
+
+    <h4>Severity Level 1 — Core functionality not available</h4>
+
+    Canonical will use continuous effort according to the service level purchased, through appropriate support engineer(s) and/or development engineer(s), to provide a work-around or permanent solution. As soon as core functionality is available, the severity level will be lowered to the new appropriate severity level.
+
+    <h4>Severity Level 2 — Core functionality severely degraded</h4>
+
+    Canonical will provide concerted efforts during the applicable business hours to provide the customer with a work-around or permanent solution. As soon as core functionality is no longer severely degraded, the severity level will be lowered to level 3.
+
+    <h4>Severity Level 3 — Standard support request</h4>
+
+    Canonical will use reasonable efforts during the applicable business hours to provide the customer with a work-around or permanent solution as soon as possible, balanced against higher severity level cases. If a work-around is provided, Canonical's support engineers will continue to work on developing a permanent resolution to the case.
+
+    <h4>Severity Level 4 — Non-urgent request</h4>
+
+    Level 4 requests include cosmetic issues, informational requests, feature requests, and similar matters. Canonical does not provide a timeline or guarantee for inclusion of any feature requests. Canonical will review each level 4 case and determine whether it is a product enhancement to be considered for a future release, an issue to be fixed in the current release or an issue to be fixed in a future release. Canonical will review and respond to information requests with a reasonable level of effort during coverage hours. Canonical may close cases representing level 4 issues after responding if Canonical believes it is appropriate to do so.
+
+    <h3>Response times</h3>
+
+    - Canonical will use reasonable efforts to respond to support requests made by the customer within the response times set forth below, based on the applicable service and severity level.
+    - “Business hours” are defined as 08:00 - 18:00 Monday - Friday local to the customer’s headquarters unless another location is agreed. All times exclude public holidays.
+
+    Table of target initial response times
```

Comment:
This used to be centred:

![image](https://user-images.githubusercontent.com/519935/50825381-d1291800-1330-11e9-96fc-bebca7cf1b2f.png)

Now isn't:

![image](https://user-images.githubusercontent.com/519935/50825410-dbe3ad00-1330-11e9-8919-264faf6ac19c.png)

I do think the latter looks objectively worse. May be worth adding some custom markup just for this item:


```suggestion
    <p style="text-align: center">Table of target initial response times</p>
```